README.md

```diff
@@ -62,3 +62,14 @@ You should see a message on startup similar to:
 2019-05-07 23:27:32 INFO plugins.lazy-loaded-driver :: Registering lazy loading driver :firebolt...
 2019-05-07 23:27:32 INFO metabase.driver :: Registered driver :firebolt (parents: #{:sql-jdbc}) 🚚
 ```
+
+## Compatibility matrix
+
+| Firebolt Version | Metabase Release | Driver Version |
+|------------------|------------------|----------------|
+| 1                | <=0.47.x         | <=1.0.13       |
+| 1                | <=0.47.x         | <=1.0.13       |
+| 2                | <=0.47.x         | 3.0.0          |
+| 1&2              | <=0.47.x         | 3.0.1          |
+| 1&2              | 0.48.x           | 3.0.3          |
+| 1&2              | 0.49.x           | 3.0.3          |
```

Comment:
Do we need this part? I think we're not using Firebolt version for customer-facing documentation. We can probably have just metabase versions and driver versions matrix.

Reply:
I believe we need. At least drivers prior 3.0.3 will not work with Metabase after 48 and it is important to mention this in documentation. All custom drivers providers do this. For example Click house.

Reply:
Please re-read the question. I'm asking about Firebolt version, which we don't advertise to the users. Metabase release X driver version makes sense here.